### PR TITLE
feat: add login functionality

### LINKED ISSUE #29: feat: setup simple login button
- Discord Auth
- GitHub Auth

### LINKED ISSUE #32: feat: setup login functionality

### PR DESCRIPTION
**Pull request by ssarkar551**

### CODE REVIEW

**Comment by ssarkar551** [comment]
added login buttons for github and discord
image

**Comment by ssarkar551** [comment]
added login button
image
image

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -8,6 +9,7 @@ import { prisma } from "../../../server/db";
 
 export const authOptions: NextAuthOptions = {
   // Include user.id on session
+
```
Comment (AyanavaKarmakar) [inline src/pages/api/auth/[...nextauth].ts]:
```suggestion
```

**Review by AyanavaKarmakar — APPROVED** [review]
LGTM

**Review by Nishith-Savla — APPROVED** [review]

**Pull request merged**